### DISCUSSION TITLE
Proposal: New mediums for "news" content

### DISCUSSION THREAD

**Discussion by agates** [Podcastindex-org/podcast-namespace#487] [Ideas]
# Proposal: New mediums for "news" content

As was discussed on [Podcasting 2.0 for March 4th 2022 Episode 76: "Pining for PodPing"](http://adam.curry.com/html/PC207620220304Podcas-W2FTrnXglvd1rnKlSp4dq0sGTR7DWR.html), there is a legitimate use case to offer news as a totally separate medium outside of other audio/video content -- to the point where the medium lends itself to dedicated applications.

Therefore, I would like to propose the addition of two new mediums to help identify this kind of content for applications without manual curation:

* `audionews` -- Describes a feed of curated news in audio form.
* `videonews` -- Describes a feed of curated news in video form.

**Comment by saerdnaer** [comment]
Why the differentiation between audio and video? Do you mean a news broadcast with several items or should the medium type only be use for a single news item?

**Comment by agates** [comment]
> Why the differentiation between audio and video?

Merely going along with the existing distinction between audio and video.

> Do you mean a news broadcast with several items or should the medium type only be use for a single news item?

I was thinking it would be several items, as with a channel dedicated to news.  Really up for discussion though as the potential applications determine what they need, but I can't think of a good argument to *expect* it to be singular (even if one wants to use it in such a manner).

Also I'm not pushing hard for this, though it would certainly provide applications a way to easily discover and/or curate news content.

**Comment by theDanielJLewis** [comment]
I think "news" is a category and not even a format.

**Comment by agates** [comment]
> I think "news" is a category and not even a format.

Medium tag isn't for formats, so you clearly don't understand it.

**Comment by theDanielJLewis** [comment]
By format, I mean stuff like interview, solo, roundtable, Q&A, etc. "News" is can be either audio or video and that term does not imply either.

So I really don't understand why we need to stuff a category name in the medium tag when podcast apps can already filter to podcasts in the "News" category.

**Comment by jamescridland** [comment]
> Medium tag isn't for formats, so you clearly don't understand it.

Alecks, if you can't be polite and collegiate here, you're not welcome. This isn't the first time you've slapped someone down because you've not been clear enough in explaining your thoughts. It's bad for this group, and bad for you. Please stop it.

> there is a legitimate use case to offer news as a totally separate medium outside of other audio/video content

What's news? The top-of-hour NPR news bulletin? A two-hour long show hosted by Tucker Carlson? A thirty minute interview with someone giving their opinions on whether Elon Musk is a good person to own Twitter? An in-depth documentary series like the Trojan Horse Affair? A former UFC wrestling commentator giving his "reckons"?

I'm not sure I understand the use-case. Ideally without being rude, would you be able to explain your thoughts to bring us up to speed?
